eventarc_gke/main.tf

```diff
@@ -0,0 +1,44 @@
+/**
+* Copyright 2023 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START terraform_eventarc_enableapi]
+# Used to retrieve project_number later
+data "google_project" "project" {
+  provider = google-beta
+}
+
+# Enable required services for GKE
```

Comment:
Since you don't use `data "google_project" "project"` in this sample, you can delete it.